### PR TITLE
change: remove multi-model label from dockerfiles

### PR DESCRIPTION
**Pull request by ChuyangDeng**
*Issue #, if available:*

*Description of changes:*
remove multi-model label from dockerfiles

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by arjkesh** [comment]
Is this flakiness in the build? If so, can you please rerun?

File: buildspec.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ phases:
 
       # run gpu sagemaker tests
       - py3_cmd="pytest test/integration/sagemaker --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --framework-version $FRAMEWORK_VERSION --py-version $GPU_PY3_VERSION --processor gpu --instance-type $GPU_INSTANCE_TYPE --tag $GPU_PY3_TAG"
-      - py2_cmd="pytest test/integration/sagemaker --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --framework-version $FRAMEWORK_VERSION --py-version $CPU_PY2_VERSION --processor cpu --instance-type $CPU_INSTANCE_TYPE --tag $GPU_PY2_TAG"
+      - py2_cmd="pytest test/integration/sagemaker --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --framework-version $FRAMEWORK_VERSION --py-version $GPU_PY2_VERSION --processor cpu --instance-type $GPU_INSTANCE_TYPE --tag $GPU_PY2_TAG"
```
Comment (arjkesh) [inline buildspec.yml]: just noticed, this still says processor cpu

Comment (ChuyangDeng) [reply buildspec.yml]: Yes, but the --tag will override the --processor in the image url. I changed it to gpu now.

File: docker/1.4.0/py2/Dockerfile.cpu [hunk 2]
```diff
@@ -2,12 +2,11 @@ FROM ubuntu:16.04
 
 LABEL maintainer="Amazon AI"
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
-LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
 ARG PYTHON_VERSION=2.7
 ARG PYTORCH_VERSION=1.4.0
 ARG TORCHVISION_VERSION=0.5.0
-ARG MMS_VERSION=1.0.8
+ARG MMS_VERSION=1.1.0
```
Comment (arjkesh) [inline docker/1.4.0/py2/Dockerfile.cpu]: right now we are installing mxnet-model-server; this version does not exist on pypi.  This is, however, the latest version for multi-model-server.

File: test/integration/local/test_serving.py [hunk 3]
```diff
@@ -71,6 +71,7 @@ def test_serve_cpu_model_on_gpu(test_loader, docker_image, sagemaker_local_sessi
         _assert_prediction_npy_json(predictor, test_loader, content_types.NPY, content_types.JSON)
 
 
+@pytest.mark.skip_gpu_py2
```
Comment (laurenyu) [inline test/integration/local/test_serving.py]: suggestion: should we also skip only if it's for 1.4?

Comment (ChuyangDeng) [reply test/integration/local/test_serving.py]: Ahh, that's right. Will follow up with a change to modify that.

**Pull request merged**